### PR TITLE
refactor:  word press plugin update to address review issues

### PR DESCRIPTION
**Pull request by deltork**
### PR Goal? 

To update the Word Press plugin to address issues raised by review

### Fixes? 
n/a

### Feedback sought? 

n/a

### Priority? 
n/a

### Tests added? 

n/a

### How to test? 
n/a

### Confidence? 

high

### Version change? 

no

### CODE REVIEW

**Comment by joanise** [comment]
It seems like we're not going to keep this approach, can we close this PR?

**Comment by deltork** [comment]
> It seems like we're not going to keep this approach, can we close this PR?

We are keeping this approach! Just abandoning the plan to put it on WordPress Plugin store.

File: packages/web-component/wordpress-plugin/read-along-web-app-loader/readme.txt [hunk 1]
```diff
@@ -1,26 +1,28 @@
...
 Tags: read-along, read along, web app loading
-Requires at least: 4.7
+Requires at least: 6.5.0
 Tested up to: 6.5.2
-Stable tag: 1.0.0
+Stable tag: 1.4.0
 Requires PHP: 7.0
 License: MIT
-License URI: https://github.com/ReadAlongs/Studio-Web/blob/main/LICENSE
+License URI: GPLv2 or later
```
Comment (joanise) [inline packages/web-component/wordpress-plugin/read-along-web-app-loader/readme.txt]: Should be MIT

**Review by joanise — APPROVED** [review]
LGTM now.

**Pull request merged**